concordium-consensus/src/Concordium/GlobalState/Persistent/Bakers.hs

```diff
@@ -36,6 +36,8 @@ import Concordium.Types.HashableTo
 import Concordium.Utils.Serialization.Put
 
 -- |A list of references to 'BakerInfo's, ordered by increasing 'BakerId'.
+-- IMPORTANT NOTE: The 'Cacheable' instance relies on this not requiring recursive caching through
+-- the use of 'EagerBufferedRef's.
```

Comment:
I've revised this comment. The point is that the datatype is always fully cached in memory, so it is fine for the 'Cacheable' instance to be trivial, and that if something changes this, then it may be necessary to reconsider how the 'Cacheable' instance is implemented.